### ISSUE TITLE
Password protect results

### ISSUE THREAD

**Issue by internetinnovations** [railslove/kickerapp#119]
Right now it's possible for everyone to change any result for any match.
This means anyone could complete destroy all logged entries.

**Comment by stephanpavlovic** [comment]
Yes it is, but it is on purpose to have the system as open and as easy usable as possible. We have daily backups if something would happen and it didn't for the last 4 years
